tracing-api/src/main/java/com/palantir/tracing/api/TraceHttpHeaders.java

```diff
@@ -39,4 +39,14 @@ public interface TraceHttpHeaders {
      */
     @Deprecated
     String ORIGINATING_SPAN_ID = "X-OrigSpanId";
+
```

Comment:
nit: let's remove the comment `/** Zipkin-compatible ... */`? `Parent-Request-Id` and `For-User-Agent` are not [B3 headers](https://github.com/openzipkin/b3-propagation)